pkg/streamingpromql/functions.go

```diff
@@ -709,3 +710,38 @@ func unaryNegationOfInstantVectorOperatorFactory(inner types.InstantVectorOperat
 	o := functions.NewFunctionOverInstantVector(inner, nil, memoryConsumptionTracker, f, expressionPosition, timeRange)
 	return operators.NewDeduplicateAndMerge(o, memoryConsumptionTracker)
 }
+
+func DoubleExponentialSmoothingFunctionOperatorFactory(args []types.Operator, memoryConsumptionTracker *limiting.MemoryConsumptionTracker, annotations *annotations.Annotations, expressionPosition posrange.PositionRange, timeRange types.QueryTimeRange, argExpressions parser.Expressions) (types.InstantVectorOperator, error) {
+	f := functions.DoubleExponentialSmoothing
+
+	functionName := "double_exponential_smoothing"
+	if len(args) != 3 || len(argExpressions) != 3 {
```

Comment:
Do we care about `argExpressions` here?

Reply:
0113d89a29707df8e0b6b53617fb739b0a15e10c